imgui_ros/Camera camera
# stop projecting beyond this range
float max_range
float constant_attenuation
float linear_attenuation
float quadratic_attenuation
# TODO(lucasw) these would be interesting to implement
# Combine the focal_distance with aov
# to make image blur scale with distance
# float focal_distance
# float aperture ?
